di-ipv-credential-issuer-stub/src/main/java/uk/gov/di/ipv/stub/cred/handlers/AuthorizeHandler.java

```diff
@@ -739,30 +735,20 @@ private SignedJWT getSignedJWT(String request, PrivateKey encryptionPrivateKey)
         }
     }
 
-    private Object getCriStubData()
-            throws UnsupportedEncodingException,
-                    com.nimbusds.jose.shaded.json.parser.ParseException {
-        JSONObject js =
-                (JSONObject)
-                        new JSONParser(MODE_JSON_SIMPLE)
-                                .parse(
-                                        AuthorizeHandler.class.getResourceAsStream(
-                                                "/data/criStubData.json"));
-
-        return js.get("data");
+    private String getCriStubData() throws IOException {
+        return OBJECT_MAPPER
+                .readTree(AuthorizeHandler.class.getResourceAsStream("/data/criStubData.json"))
+                .get("data")
+                .toString();
     }
 
-    private Object getCriStubEvidencePayloads()
-            throws UnsupportedEncodingException,
-                    com.nimbusds.jose.shaded.json.parser.ParseException {
-        JSONObject js =
-                (JSONObject)
-                        new JSONParser(MODE_JSON_SIMPLE)
-                                .parse(
-                                        AuthorizeHandler.class.getResourceAsStream(
-                                                "/data/criStubEvidencePayloads.json"));
-
-        return js.get("data");
+    private Object getCriStubEvidencePayloads() throws IOException {
+        return OBJECT_MAPPER
+                .readTree(
+                        AuthorizeHandler.class.getResourceAsStream(
+                                "/data/criStubEvidencePayloads.json"))
+                .get("data")
+                .toString();
```

Comment:
Should this method and above return `String` if we know it will be one?

Reply:
Oh, I updated one and not the other. Will fix up